pages/honoraryMembers/index.tsx

```diff
@@ -0,0 +1,101 @@
+import type { NextPage } from "next";
+import React, { useState } from "react";
+import Header from "../../components/Header";
+import styles from "../../styles/Member.module.css";
+import { Stack, Button } from "@mui/material";
+import HonoraryMember from "../../components/honoraryMember";
+import { members, info } from "../../data/membersUtils";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Typography from "@mui/material/Typography";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import cx from "classnames";
+
+const HonoraryMembersPage: NextPage = () => {
+  const [pageDisplay, setPageDisplay] = useState(true);
+
+  const AreMembers = members.map((m) => <HonoraryMember key={m.name} {...m} />);
+  const displayAreMembers = (
```

Comment:
This is still a mix between norwegian and english, change it to f.ex. `HonoraryMembers`

Reply:
I think its all changed now

Reply:
great, but there is a small typo, in `displayhonoraryMembers` the "h" should be upper case

Reply:
i apologize